Instagrid/Instagrid/View/ImagesOrganizer.swift

```diff
@@ -0,0 +1,80 @@
+//
+//  ImagesOrganizer.swift
+//  Instagrid
+//
+//  Created by Elsa Jeannin on 02/12/2018.
+//  Copyright © 2018 Elsa Jeannin. All rights reserved.
+//
+
+import UIKit
+
+class ImagesOrganizer: UIView {
```

Comment:
Small detail, since this class is an UIView I think it would be better to name it `ImagesOrganizerView` WDYT?